src/landing-page/landing-page.js

```diff
@@ -0,0 +1,300 @@
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import {
+	useState,
+	useEffect,
+	createInterpolateElement,
+} from '@wordpress/element';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import {
+	Button,
+	ExternalLink,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalVStack as VStack,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalHStack as HStack,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { downloadExportedTheme } from '../resolvers';
+import { downloadFile } from '../utils';
+import { CreateThemeModal } from './create-modal';
+
+export default function LandingPage() {
+	const [ themeName, setThemeName ] = useState( '' );
+	const [ createModalType, setCreateModalType ] = useState( false );
+
+	const currentTheme = useSelect( ( select ) =>
+		select( coreStore ).getCurrentTheme()
+	);
+
+	useEffect( () => {
+		const name = currentTheme?.name?.raw || '';
+		setThemeName( name );
+	}, [ currentTheme ] );
```

Comment:
As @t-hamano mentioned, it's not necessary to manage the `themeName` as a state variable.

```suggestion
export default function LandingPage() {
	const [ createModalType, setCreateModalType ] = useState( false );

	const themeName = useSelect( ( select ) =>
		select( coreStore ).getCurrentTheme()
	)?.name?.raw;
```

Reply:
Ah, yes of course.